dev-env.yaml

```diff
@@ -30,9 +30,11 @@ services:
     ports:
       - "5432:5432"
     environment:
-      POSTGRES_DB: reportvision
-      POSTGRES_USER: postgres
```

Comment:
We might need to make a separate app.yaml file but this is something we can handle as part of next steps. This looks ok!